internal/pkg/utils/strhelper/strhelper.go

```diff
@@ -96,7 +96,7 @@ func NormalizeName(str string) string {
 	// Replace special characters with one separator
 	// "---Camel-Case" -> "-Camel-Case"
 	str = regexpcache.
-		MustCompile(`[^a-zA-Z0-9]+`).
+		MustCompile(`[^a-zA-Zá-žÁ-Ž0-9]+`).
```

Comment:
Doufám teda že jsem pochopil dobře, že chceme znaky s diakritikou v názvech povolit. 😄

Reply:
skor to chceme asi previes to ascii? napr. `á` -> `a` ... regexp je ok, staci pridat este jeden riadok, ktory odstrani diakritiku, alebo ti pride lepsie mat tam diakritiku? trocha sa obavam ci to pojde bez problemov na vsetkych platformach, a osobne by som tam diakritiku mat nechce (ako uzivate).

Reply:
http://blog.golang.org/normalization

https://stackoverflow.com/a/26722698

Reply:
Aha, jasně. 🙂 No mně to přišlo nějaký podezřelý. 😅